assets/src/components/form/layouts/two-column-admin/two-column-admin.css

```diff
@@ -0,0 +1,390 @@
+/* BOOTSTRAP Styles
+* copied here to prevent them affecting entire WordPress admin
+* -------------------------------------------------------------------------*/
+
+.ee-two-column-admin {
+    margin: 0 !important;
+    padding: 0 !important;
+}
```

Comment:
mmmmm... ya... hard to say at his point... I'd be inclined to cross bridges like that as we get to them... it would just be too difficult at this point to try abstracting things like that without some concrete use cases to work with. 

I definitely think this is a good idea and is something I would like to implement.

If anything... I'm tempted to push for utilizing a solid and mature third party library for components and layout, specifically the [Material UI library](https://material-ui.com/getting-started/supported-components/), which has the benefit of *years* of development, support by one of the biggest companies out there, rock solid accessibility (no dig intended at Gutenberg... wait... what am i saying? ...that totally IS intended to be a dig at GB!), and is *way* more customizable and extensible than the Gutenberg components (which are pretty crappy and buggy so far imho).

I especially like how easy it is to implement theming with the Material UI.

Reply:
Ya I like the Material UI as well.  I'm mostly concerned with the contrast between it and the WP interface.  Regardless of how better designed it is, will it stand out in the WordPress UI (in a negative way)?

Reply:
> Some are still extremely buggy and we know that accessibility is not at an ideal state yet. 

This is a bit hyperbole.  Like any project there are bugs, but I would not consider the GB components as a whole to be "extremely buggy".   

Regarding accessibility, I  know that resources ARE being put towards accessibility for the components.  The accessibility drama you've been hearing about with regards to GB is not really that no work is being done on accessibility, but rather mostly that accessibility was not included from the get go in GB design.  On the whole, components are getting accessibility treatment (just recently a completely reworked, accessible datepicker was merged in).  

While again I _like_ the Material UI, I'm wary about adopting something wholesale for the WP admin that _could_ provide a jarring experience for users.  That doesn't mean we shouldn't experiment and see if that's actually the case, but I don't want to disregard the great work already done by smart folks on components for use within the WP context (which is only going to grow in availability once WP 5.0 ships).